api/c/indigo/src/indigo_structure_checker.h

```diff
@@ -43,8 +43,8 @@ namespace indigo
         CheckResult check(const IndigoObject& item, const std::vector<CheckTypeCode>& check_types = std::vector<CheckTypeCode>(),
                           const std::vector<int>& selected_atoms = std::vector<int>(), const std::vector<int>& selected_bonds = std::vector<int>());
 
-        std::string toJson(const StructureChecker2::CheckResult& res);
-        std::string toJson2(const StructureChecker2::CheckResult& res);
+        std::string toJson(const StructureChecker::CheckResult& res);
+        std::string toJson2(const StructureChecker::CheckResult& res);
```

Comment:
Do we still need both `toJson` and `toJson2` methods?

Reply:
fixed